ansible/roles/test/tasks/fdb_mac_expire.yml

```diff
@@ -0,0 +1,109 @@
+- fail: msg="testbed_type is not defined"
+  when: testbed_type is not defined
+
+- fail: msg="testbed_type {{test_type}} is invalid"
+  when: testbed_type not in ['t0', 't0-64', 't0-64-32', 't0-116']
+
+- include_vars: "vars/topo_{{testbed_type}}.yml"
+
+- name: Expand properties into props
+  set_fact: props="{{configuration_properties['common']}}"
+
+- name: Gather minigraph facts about the device
+  minigraph_facts: host={{inventory_hostname}}
+
+- name: Copy tests to PTF
+  copy: src=roles/test/files/ptftests dest=/root
+  delegate_to: "{{ptf_host}}"
+
+- name: Copy FDB information file to PTF
+  template: src=roles/test/templates/fdb.j2 dest=/root/fdb_info.txt
+  delegate_to: "{{ ptf_host }}"
+
+- name: clear fdb all
+  shell: clear fdb all
```

Comment:
The right command for clearing fdb should be `sonic-clear fdb all`.

Reply:
I think, clear is mapped to sonic-clear when used with arg. 
Else https://github.com/Azure/sonic-utilities/pull/269 would not had beeen applicable.

Reply:
I double checked, mapping is not present in github. Within Linkedin, we still map clear to /usr/bin/clear without arg and to sonic-clear with arg:


admin@falco-test-dut01:~$ clear ?
Usage: sonic-clear [OPTIONS] COMMAND [ARGS]...

  SONiC command line - 'Clear' command

Options:
  -?, -h, --help  Show this message and exit.

Commands:
  arp            Clear IP ARP table
  bgp            BGP information
  counters       Clear counters
  fdb            Clear FDB table
  ip             Clear IP
  ipv6           Clear IPv6 information
  ndp            Clear IPv6 NDP table
  pfccounters    Clear pfc counters
  queuecounters  Clear queue counters
admin@falco-test-dut01:~$

Reply:
I am not sure whether all vendor's platforms have this mapping. At least this mapping is not available on Mellanox platform. BTW, `clear` also conflicts with the linux builtin command for clearing terminal screen display.
It would be better to use the original command `sonic-clear` which is guranteed available on all platforms.

Reply:
It fails with "clear". Please correct this to use "sonic-clear"